AI/pyneogame/Agent/PolicyGradient.py

```diff
@@ -232,3 +251,10 @@ def replay_experience(self, batch_size=64, epochs=30):
                                      )
 
         return history
+    
+    def input_model(self, model):
```

Comment:
Take the model as input to the constructor instead and use the same model creation pattern as done in dqn_test.py